.github/workflows/integration-test.yml

```diff
@@ -71,14 +71,18 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: instill-ai/instill-core
+          ref: jvalles/ins-6570-pass-oauth-client-and-secret-ids-to-backend-and-frontend # TODO remove
```

Comment:
Hi @donch1989 can you have a look at [this](https://github.com/instill-ai/instill-core/pull/1143), too? If it looks OK I'll merge that and remove this line